testing/tests/test_transformers.py

```diff
@@ -0,0 +1,105 @@
+import filecmp
+import json
+import perftestnotebook
+import pytest
+import yaml
+from utilities import get_nested_values
+from functools import reduce
+from operator import add
+from pathlib import Path
+
+
+class TestTransformer(object):
+    def check_transformation(self, test_config_filepath, target_config_filepath, get_nested_keys):
+        """
+        This function checks:
+            - Both of test and local config files from the arguments have "file_groups" field.
+            - Each entry of the transformed output has "data", "name", and "subtest" fields.
+                - Each "data" field has "file", "value", and "xaxis" fields.
+                    - Each file from "file" field is included in the "file_groups" field of the config file.
```

Comment:
These lines seem to be longer than 100 columns, can you check if it passes black formatting?

Reply:
It passes black formatting. I guess black does not format comments. Now I modify them to less than 100 columns manually.